maven-release-manager/src/main/java/org/apache/maven/shared/release/util/MavenCrypto.java

```diff
@@ -63,6 +63,9 @@ private MavenCryptoException(String message, Throwable cause) {
     public MavenCrypto(DefaultSecDispatcher secDispatcher, PlexusCipher plexusCipher) {
         this.secDispatcher = secDispatcher;
         this.plexusCipher = plexusCipher;
+
+        // Adjust the default path (def path != maven path)
+        this.secDispatcher.setConfigurationFile("~/.m2/settings-security.xml");
```

Comment:
Does this work? I'm not sure. Windows and even on unix system?
It would be better to use something such `System.getProperty("user.home")+"/.m2/settings-security.xml"`

Reply:
It does, sec dispatcher resolves this for us.

Reply:
https://github.com/codehaus-plexus/plexus-sec-dispatcher/blob/0e1eefd2eefcee9cfbd9a8fdca22e01cfe3ead55/src/main/java/org/sonatype/plexus/components/sec/dispatcher/DefaultSecDispatcher.java#L224-L227

Reply:
ditto. I wish Java could do it as well :) 
well maybe a comment as it is not very intuitive. Image when some archeologists will have a look at this code with the new Maven dependencies cleanup in few years